config/owasp/suppressions.xml

```diff
@@ -102,6 +118,27 @@
     <packageUrl regex="true">^pkg:maven/org\.bouncycastle/bcprov\-jdk15on@.*$</packageUrl>
     <vulnerabilityName>CVE-2023-33201</vulnerabilityName>
   </suppress>
+  <suppress>
```

Comment:
Check if these suppressions are still needed

Reply:
Looks like they are still needed. There doesn't seem to be a fix for the first one and the second two require azure migration in et-common.